src/data/achievements.js

```diff
@@ -275,6 +275,18 @@ const achievements = [
     condition: state => state.revenue >= goal,
     reward: state => state,
   }))(),
+
+  (() => ({
+    id: 'gold-digger',
+    name: 'Gold Digger',
+    description: `Pay Off Your Loan.`,
+    rewardDescription: `A Shovel to use to dig for ores in your field`,
```

Comment:
This content is all the right format, but subjectively feels imbalanced compared to how the other achievements look. Here's a comparison: 

<img width="383" alt="Screen Shot 2021-07-13 at 8 40 33 AM" src="https://user-images.githubusercontent.com/366330/125462031-17569b58-3532-45d3-974f-c072ca7b6243.png">

I feel that most of the content here should be in the `description` and that `rewardDescription` should be as brief as possible. Here's an example of what I'm getting at: https://github.com/jeremyckahn/farmhand/blob/cb4734a9727f2268b9cd3cbc9a1ac70a9d6d8510/src/data/achievements.js#L70-L83

This makes for a more "bottom heavy" text display, which is more consistent with the other achievements, if nothing else. Do you think this could be reworded a bit to fit in with the other achievements a bit?

Reply:
ah, yep. i think i did this same sort of "backwards" thing when i first did the i am rich implementation too 😄 . will update